requirements.txt

```diff
@@ -38,3 +38,4 @@ CJKwrap==2.2
 typing-extensions==4.10.0
 uvloop==0.19.0; sys_platform!='win32' and sys_platform!='cygwin' and sys_platform!='cli'
 isal==1.6.0; platform_machine=='x86_64' or platform_machine=='AMD64' or platform_machine=='aarch64'
+requests==2.26.0
```

Comment:
RSStT is asynchronous, using synchronous HTTP lib will make the bot stuck.
Use the `web` wrapper (in `src/web/req.py`) instead.